server/middleware/auditMiddleware.ts

```diff
@@ -0,0 +1,76 @@
+import { NextFunction, Request, Response, Router } from 'express'
+import { Services } from '../services'
+import { Page, PageViewEventDetails } from '../services/auditService'
+import asyncMiddleware from './asyncMiddleware'
+
+const pageViewEventMap: Record<string, Page> = {
+  '/': Page.PRISONER_LIST,
+  '/plan/:prisonNumber/view/overview': Page.OVERVIEW,
+  '/plan/:prisonNumber/view/support-needs': Page.SUPPORT_NEEDS,
+  '/plan/:prisonNumber/view/education-and-training': Page.EDUCATION_AND_TRAINING,
+  '/plan/:prisonNumber/view/work-and-interests': Page.WORK_AND_INTERESTS,
+  '/plan/:prisonNumber/view/timeline': Page.TIMELINE,
+  '/plan/:prisonNumber/functional-skills': Page.FUNCTIONAL_SKILLS,
+  '/plan/:prisonNumber/goals/create': Page.CREATE_GOALS,
+  '/plan/:prisonNumber/goals/:goalIndex/add-step/:stepIndex': Page.CREATE_GOAL_ADD_STEP,
```

Comment:
Could you add a todo and perhaps a bit of spacing between lines 15 and 18, because these routes will be removed soon(ish) as part of migrating to the single screen journey for creating goals. We already have a jira to remove the feature toggle and associated "old" code, so these audit routes would fall within that remit.
Perhaps something like:
```
 ...
 '/plan/:prisonNumber/functional-skills': Page.FUNCTIONAL_SKILLS,
  '/plan/:prisonNumber/goals/create': Page.CREATE_GOALS,

  // TODO - RR-769 - Remove route auditing for original Create Goal journey
  '/plan/:prisonNumber/goals/:goalIndex/add-step/:stepIndex': Page.CREATE_GOAL_ADD_STEP,
  '/plan/:prisonNumber/goals/:goalIndex/add-note': Page.CREATE_GOAL_ADD_NOTE,
  '/plan/:prisonNumber/goals/:goalIndex/create': Page.CREATE_GOAL,
  '/plan/:prisonNumber/goals/review': Page.CREATE_GOALS_REVIEW,

  '/plan/:prisonNumber/goals/:goalReference/update': Page.UPDATE_GOALS,
  '/plan/:prisonNumber/goals/:goalReference/update/review': Page.UPDATE_GOALS_REVIEW,
   ...
```